mobile/src/main/java/rs/readahead/washington/mobile/media/MediaFileHandler.java

```diff
@@ -199,16 +200,24 @@ public static void exportMediaFile(Context context, VaultFile vaultFile, @Nullab
 
     public static Single<VaultFile> importPhotoUri(Context context, Uri uri, @Nullable String parentId) throws Exception {
         // Vault replacement methods
-        InputStream v_input = context.getContentResolver().openInputStream(uri); // original photo
-        Bitmap v_bm = modifyOrientation(BitmapFactory.decodeStream(v_input), v_input); // bitmap of photo
-        Bitmap v_thumb = ThumbnailUtils.extractThumbnail(v_bm, v_bm.getWidth() / 10, v_bm.getHeight() / 10); // bitmap of thumb
-
+        Boolean keepExif = Preferences.isKeepExif();
```

Comment:
Let's  change `Boolean` to `boolean` for the keepExif variable, as it is a primitive type.